test/test_issue325.js

```diff
@@ -0,0 +1,66 @@
+
+exports.test = function(sql, assert){
+    "use strict";
+    // Create a database
+    var db = new sql.Database();
+    var result;
+    var stmt;
+    var value;
+
+    // inline result value test
+    result = db.exec("SELECT 1.7976931348623157e+308 AS myResult");
+    assert.strictEqual(result.length, 1, 'correct inline result length');
+    assert.deepEqual(result[0].columns, ['myResult'], 'correct inline result column name');
+    assert.strictEqual(result[0].values.length, 1, 'correct inline result values length');
+    assert.strictEqual(result[0].values[0].length, 1, 'inline result values[0] length');
+    assert.strictEqual(typeof result[0].values[0][0], 'number', 'correct inline result value is a number');
+    assert.ok(result[0].values[0][0] > 1.7976931348623e+308, 'inline result value is not too small');
+    assert.ok(result[0].values[0][0] < 1.797693134862316e+308, 'inline result value is not too large');
```

Comment:
Wasn't the goal of this PR to finally have an exact result ? If so, then we can test that result === 1.7976931348623157e+308